hajji/hello.S

```diff
@@ -61,6 +67,7 @@ start:
         callw clrscr                  # clear screen
         callw curshome                # move cursor home - top:left
         callw greeting                # display a greeting string
+		cli
```

Comment:
Copy and paste without removing the original. I originally had it outside the comment and intended to paste it below the comment after but didn't remove the original.